README.md

```diff
@@ -1244,8 +1244,7 @@ _You can enable the following settings in Xcode by running [this script](resourc
     at: location,
     count: 5,
     color: starColor,
-    withAverageDistance: 4
-  )
+    withAverageDistance: 4)
```

Comment:
The `Put the closing parenthesis on the last line of the invocation.` sentence at L1228 needs adjustment as well, I think